awx/main/migrations/0124_sysjob_default_days.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+
+from django.db import migrations, models
+
+from awx.main.utils.common import set_current_apps
+from awx.main.models import SystemJobTemplate
+
+
+def set_default_days(apps, schema_editor):
+    set_current_apps(apps)
+    for sys_template in SystemJobTemplate.objects.all():
+        if sys_template.has_configurable_retention:
+            if sys_template.default_days is None:
+                sys_template.default_days = 30
```

Comment:
We used to hard-code the default of `30` in the old ui, but now this field lives in the api. System jobs with configurable retention are assigned this value on migration.